boomerang.js

```diff
@@ -52,6 +52,16 @@
  * otherwise, it is `Date.now()`.
  */
 
+ /**
+ * @function
+ * @global
+ * @desc
+ * Checks if we are working with IE version 8 and lower.
+ */
+function isIe8OrLower(){
+	return !document.getElementsByClassName;
+}
```

Comment:
The function should probably not be here, will look for a better place for it.

Reply:
Yeah this would leak to the global scope... maybe we can construct `BOOMR` here earlier?

```
if (!window.BOOMR) {
    window.BOOMR = {};
}

window.BOOMR.isIe8OrLower = function() { ... }
```

Or, since this is such a simple check, we could just put `!document.getElementsByClassName` in all of those places.

Reply:
Keep in mind that this test will fail on pages that use a `getElementsByClassName` polyfill, so it's not sufficient.  You'll need to check that it exists, and that it is a function, and that it is a native function.

Reply:
@nicjansma @bluesmoon All good points, thank you 👍 Let me spend some time thinking about this and, if time allows, I'll propose a solution.